pkg/doc/sdjwt/integration_test.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package sdjwt
+
+import (
+	"crypto/ed25519"
+	"crypto/rand"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	afjwt "github.com/hyperledger/aries-framework-go/pkg/doc/jwt"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/sdjwt/holder"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/sdjwt/issuer"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/sdjwt/verifier"
+)
+
+const (
+	testIssuer = "https://example.com/issuer"
+)
+
+func TestSDJWTFlow(t *testing.T) {
+	r := require.New(t)
+
+	pubKey, privKey, e := ed25519.GenerateKey(rand.Reader)
+	r.NoError(e)
+
+	signer := afjwt.NewEd25519Signer(privKey)
+	claims := map[string]interface{}{
+		"given_name": "Albert",
+		"last_name":  "Smith",
+	}
+
+	signatureVerifier, e := afjwt.NewEd25519Verifier(pubKey)
+	r.NoError(e)
+
+	t.Run("success", func(t *testing.T) {
+		// Issuer will issue SD-JWT for specified claims.
+		token, e := issuer.New(testIssuer, claims, nil, signer)
+		r.NoError(e)
+
+		// TODO: Should we have one call instead of two (designed based on JWT)
+		sdJWTSerialized, e := token.Serialize(false)
+		r.NoError(e)
+
+		fmt.Println(fmt.Sprintf("issuer SD-JWT: %s", sdJWTSerialized))
```

Comment:
is this `fmt.println()` leftover?

Reply:
No, I left print outs on purpose to show output formats.